main/src/modules/systemVisual/iol2opc/src/module.cpp

```diff
@@ -808,8 +816,10 @@ void IOL2OPCBridge::updateOPC()
         if (!unknownObjectInScene)
             onlyKnownObjects.heartBeat();
 
+        objLocOut.write(bStreamObjLoc);
```

Comment:
Very true, I haven't thought about this. Can you please change it @maxime-petit?